scripts/functions.sh

```diff
@@ -205,9 +206,87 @@ function setup {
     ${K8S_CLI_BIN} rollout status daemonset netobserv-cli -n netobserv-cli --timeout 60s
     rm -rf ${MANIFEST_OUTPUT_PATH}
   elif [ "$1" = "packets" ]; then
+    shift
     echo "creating packet-capture agents"
-    echo "${packetAgentYAML/"{{PCA_FILTER_VALUE}}"/${2:-}}" | ${K8S_CLI_BIN} apply -f -
+    if [[ ! -d ${MANIFEST_OUTPUT_PATH} ]]; then
+      mkdir -p ${MANIFEST_OUTPUT_PATH} > /dev/null
+    fi
+    manifest="${MANIFEST_OUTPUT_PATH}/${PACKETS_MANIFEST_FILE}"
+    echo "${packetAgentYAML}" > ${manifest}
+    options="$*"
+    # Iterate through the command-line arguments
+    for option in $options; do
+        key="${option%%=*}"
+        value="${option#*=}"
+        case "$key" in
+            --enable_pca) # Enable packet capture agent
+                if [[ "$value" == "true" || "$value" == "false" ]]; then
+                  edit_manifest "pca_enable" "$value" "$manifest"
+                else
+                  echo "invalid value for --pca_enable"
+                fi
+                ;;
```

Comment:
Not sure to get the point of this option.

To run a packet capture, the user is supposed to type `oc netobserv packets` right ?

Reply:
Yes the above default is true the only use of this as way to disable pca which I don’t see a use for it in production see the new doc

Reply:
I will drop this option